src/Commands/CommandBus.php

```diff
@@ -204,12 +204,12 @@ protected function handler(Update $update): Update
     {
         $message = $update->getMessage();
 
-        if ($message !== null && $message->has('text') && $message->has('entities')) {
-
+        if ($message->has('entities')) {
```

Comment:
That totally makes sense! Returning empty collection. We need to do this to all other methods where it's simply null. Because it ends up failing all `has` and other methods.